packages/virtualized-lists/Lists/VirtualizedListCellRenderer.js

```diff
@@ -8,13 +8,15 @@
  * @format
  */
 
-import type {ViewStyleProp} from '../StyleSheet/StyleSheet';
-import type {FocusEvent, LayoutEvent} from '../Types/CoreEventTypes';
+import type {ViewStyleProp} from 'react-native/Libraries/StyleSheet/StyleSheet';
+import type {
+  FocusEvent,
+  LayoutEvent,
+} from 'react-native/Libraries/Types/CoreEventTypes';
```

Comment:
TIL these Flow types are not exported. They are public in TypeScript though. We should probably be re-exporting types from `CoreEventTypes`, but that might be a good followup.